src/wp-includes/class-wp-block-patterns-registry.php

```diff
@@ -79,6 +79,7 @@ final class WP_Block_Patterns_Registry {
 	 *                                   of the post types passed on the array. For all the other post types
 	 *                                   not part of the array the pattern is not available at all.
 	 *     @type string[] $templateTypes Optional. An array of template types where the pattern fits.
+	 *     @type string   $file_path     Optional. The path to the file containing the block pattern content.
```

Comment:
Is this the full path, relative path, etc? Can we be more specific?

Reply:
Can we also update `*     @type string   $content       Required. Block HTML markup for the pattern.` so denote that is not required.

Reply:
@joemcgill @jonnynews  I have addressed both.